app/src/main/java/com/squirtles/musicroad/create/CreatePickScreen.kt

```diff
@@ -80,20 +91,55 @@ fun CreatePickScreen(
 
     Log.d("CreatePickScreen", song.toString())
 
-    CreatePickDisplay(
-        song = song,
-        comment = comment.value,
-        dynamicBackgroundColor = dynamicBackgroundColor,
-        dynamicOnBackgroundColor = dynamicOnBackgroundColor,
-        onBackClick = {
-            createPickViewModel.resetComment()
-            onBackClick()
-        },
-        onCreateClick = {
-            createPickViewModel.createPick(onCreateClick)
-        },
-        onCommentChange = createPickViewModel::onCommentChange
-    )
+    // 생성 중 인디케이터가 표시되고 있을 때는 시스템의 뒤로 가기 버튼 클릭을 무시
+    BackHandler(enabled = showCreateIndicator) { }
+
+    when (uiState) {
+        CreateUiState.Default -> {
+            CreatePickDisplay(
+                song = song,
+                comment = comment.value,
+                dynamicBackgroundColor = dynamicBackgroundColor,
+                dynamicOnBackgroundColor = dynamicOnBackgroundColor,
+                onBackClick = {
+                    createPickViewModel.resetComment()
+                    onBackClick()
+                },
+                onCreateClick = {
+                    createPickViewModel.onCreatePickClick()
+                    showCreateIndicator = true
+                },
+                onCommentChange = createPickViewModel::onCommentChange
+            )
+        }
+
+        is CreateUiState.Success -> {
+            showCreateIndicator = false
+            val pickId = (uiState as CreateUiState.Success<String>).data
+            onCreateClick(pickId)
+        }
+
+        CreateUiState.Error -> {
+            // TODO()
+            showCreateIndicator = false
+            Text("생성 오류")
+        }
+    }
+
+    if (showCreateIndicator) {
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(Black.copy(alpha = 0.5F))
+                .clickable( // 클릭 효과 제거 및 클릭 이벤트 무시
+                    interactionSource = remember { MutableInteractionSource() },
+                    indication = null
+                ) { },
```

Comment:
```suggestion
                .clickable( // 클릭 효과 제거 및 클릭 이벤트 무시
                    interactionSource = remember { MutableInteractionSource() },
                    indication = null,
                    onClick = {}
                ),
```

onClick이 비어있는 것을 이렇게 나타내는 것이 더 알기 쉬울 것 같습니다!